src/main/webapp/WEB-INF/web.xml

```diff
@@ -75,5 +75,10 @@
     <servlet-name>ActivityServlet</servlet-name>
     <url-pattern>/activity.jsp</url-pattern>
   </servlet-mapping>
+  
+   <servlet-mapping>
+    <servlet-name>ProfileServlet</servlet-name>
+    <url-pattern>/profile.jsp</url-pattern>
```

Comment:
@VillanuevaK @lucieleblanc @laurenfraser2 

I just noticed with the existing <url-pattern>
<url-pattern>/register</url-pattern>
or
<url-pattern>/login</url-pattern>

There is NO '.jsp' at the end like in
<url-pattern>/activity.jsp</url-pattern>

I am not sure what the significance of this difference is. But something to keep in mind.

Reply:
The jsp has no significance, but it is probably better redirect the user to the /activity page will get on that in future pull request.